Linear/KirchhoffLovePlate.h

```diff
@@ -244,6 +248,9 @@ class KirchhoffLovePlateNorm : public NormBase
   //! \brief Empty destructor.
   virtual ~KirchhoffLovePlateNorm() {}
 
+  //! \brief Defines the Neumann order that is the subject of integration.
+  virtual void setNeumannOrder(char flag) { nOrder = flag; }
```

Comment:
No, it is a virtual method in `Integrand`, such that it can be accessed via that interface in the ASM-methods.